recipes/dcase2021_task4_separation_SED/local/sed_training.py

```diff
@@ -0,0 +1,260 @@
+import os
+import torch
+import pytorch_lightning as pl
+from collections import OrderedDict
+import pandas as pd
+from .utils import batched_decode_preds, log_sedeval_metrics
+
+
+class DESED(pl.LightningModule):
+    def __init__(
+        self,
+        hparams,
```

Comment:
@turpaultn actually however there is an advantage in having the yaml. It can be saved in the experiment directory and then it is easy to load to restore exactly the same hyperparameters. It is not so easy to do so with python. You have to import that.

Reply:
Well, it's just a matter of organising the python class, we could add a "save" function that saves the parameters in the yaml format, and a load to load from the yaml file.
Actually, even for the yaml file we have now, it could be a good solution because the python class could be helpful to keep track of the "used" parameters --> We could imagine the dictionary being filled when we do a "get" from the Configuration class, so all the unused parameters are not saved unnecessarily, what do you think ?